src/Core/Validation/FieldValidation/DecimalFieldValidator.php

```diff
@@ -3,7 +3,7 @@
 namespace SilverStripe\Core\Validation\FieldValidation;
 
 use SilverStripe\Core\Validation\ValidationResult;
-use SilverStripe\Core\Validation\FieldValidation\NumericFieldValidator;
+use SilverStripe\Core\Validation\FieldValidation\NumericNonStringFieldValidator;
```

Comment:
```suggestion
```
Shouldn't be needed since it's in the same namespace

Reply:
Updated